src/app/components/popup-choose-the-time/popup-choose-the-time.component.html

```diff
@@ -1,7 +1,7 @@
 <app-modal-window [title]="title">
   <div class="wrapper-content">
-    <div class="wrapper-data" [title]="titleData">
-      <span class="title-data">{{ titleData }}</span>
+    <div class="wrapper-data">
+      <span class="title-data">{{ selectDate }}</span>
```

Comment:
would u add translation here? or this is date?

Reply:
we won’t use this popup in the calendar